app/assets/javascripts/discourse/app/controllers/user-status.js

```diff
@@ -0,0 +1,56 @@
+import Controller from "@ember/controller";
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+import { action } from "@ember/object";
+import { notEmpty } from "@ember/object/computed";
+import { inject as service } from "@ember/service";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import bootbox from "bootbox";
+
+export default Controller.extend(ModalFunctionality, {
+  userStatusService: service("user-status"),
+
+  description: "",
```

Comment:
Im usually not a big fan of empty strings, is there a reason to prefer this over `null` ?

Reply:
The usual argument is that it is usually easier to deal with a single data type `String` instead of having to deal with two different data types.

Reply:
Yep agreed on backend, but I don't think that works well in frontend code. As this example shows you end up having a lot of `if (foo.length)` also as this is not type you are never sure if it's `null` or `""` so you actually use something like `foo?.length` or any `not empty` kind method.

Reply:
I think I more agree with @jjaffeux here. I switched it to using `null`, it makes code simpler. 

But soon we may decide to switch it back to an empty string 😄. The reason is that when we add emoji picker to the status modal, we're going to allow users to set status that contains only emoji. It may turn out that an empty string will  be more convenient at that moment. But, we'll see when implementing that.